source/features/highlight-deleted-and-added-files-in-diffs.tsx

```diff
@@ -39,7 +39,7 @@ async function init(): Promise<void | false> {
 			filename.classList.add('rgh-pr-file-state');
 			const sourceIcon = pageDetect.isPR() ?
 				select(`[href="${filename.hash}"] svg`, fileList)! :
-				select(`svg + [href="${filename.hash}"]`, fileList)?.previousElementSibling!;
+				select(`svg + [href="${filename.hash}"]`, fileList)?.previousElementSibling as SVGSVGElement;
```

Comment:
SVG elements don't have `href` attributes, this is not `<SVG>` being selected here. Is the selector wrong?